### ISSUE TITLE
Improve overriding dimens in SDC

### ISSUE THREAD

**Issue by FikriMilano** [google/android-fhir#1832]
**Is your feature request related to a problem? Please describe.**
In client app, I want to override the dimens property to change the margin or padding of the UI components. But if I change a dimens property that is used in the component I like to change, it also affects the other components. Because those dimens property is used in both components. 

e.g. padding_default is used as margin top of widgets and padding of check box, overriding this property will affect both, which I don't want to happen.

**Describe the solution you'd like**
Overriding a dimens property should only affect the component people intend to change.

**Describe alternatives you've considered**
N/A

**Additional context**
This is related to the UX Questionnaire adjustments that is ongoing on FHIRCore https://github.com/opensrp/fhircore/issues/1974.

Current design =================> Adjusted design
<img width="250" alt="image" src="https://user-images.githubusercontent.com/62053304/215712886-eb7239d9-2d8e-42f2-a652-98a4e6d7e8c1.png">   <img width="238" align="top" alt="image" src="https://user-images.githubusercontent.com/62053304/215713121-410eb84a-43fd-4a67-8781-2e955c448a5e.png">

In summary of our UX changes that depends on SDK:
- Reduce margin between question and body, should be 16px
- Add more margin between widgets, should be 32px
- Reduce margin between selection options, should be 8px

**Would you like to work on the issue?**
Yeah

**Comment by FikriMilano** [comment]
@shelaghm @f-odhiambo

**Comment by FikriMilano** [comment]
@jingtang10 are you going to work on this? or should I?

**Comment by jingtang10** [comment]
@santosh-pingle please take a look at this PR!

**Comment by jingtang10** [comment]
@shelaghm fyi
